src/main/scala/Errors/DateFormatError.scala

```diff
@@ -0,0 +1,3 @@
+package Errors
+final case class DateFormatError(private val message: String = "",
```

Comment:
Do you need to specify that these are private properties? Can't we just treat them like local variables instead of class properties?

Reply:
Done